tools/cldr-apps/src/main/webapp/js/special/users.js

```diff
@@ -69,7 +69,7 @@ define("js/special/users.js", ["js/special/SpecialPage.js"], function (
         var locked = [];
 
         var byEmail = {}; // email -> u:{}
-        var byId = {}; // id -> u:{}
+        var byId = {}; // id -> u:{} -- UNUSED!
```

Comment:
drop if unused

Reply:
that whole file will be unused when we drop dojo -- I just added the comment as a reminder during the transition